### PR TITLE
Feature/integration nebius

### PR DESCRIPTION
**Pull request by Aktsvigun**
# Description

Nebius AI Studio provides access to state-of-the-art AI models.

Fixes # (issue)

## New Package?

Did I fill in the `tool.llamahub` section in the `pyproject.toml` and provide a detailed README.md for my new integration or package?

- [X] Yes
- [ ] No

## Version Bump?

Did I bump the version in the `pyproject.toml` file of the package I am updating? (Except for the `llama-index-core` package)

- [ ] Yes
- [X] No

## Type of Change

Please delete options that are not relevant.

- [ ] Bug fix (non-breaking change which fixes an issue)
- [X] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)
- [ ] This change requires a documentation update

## How Has This Been Tested?

Your pull-request will likely not be merged unless it is covered by some form of impactful unit testing.

- [X] I added new unit tests to cover this change
- [X] Added new notebooks (that test end-to-end)
- [ ] I believe this change is already covered by existing unit tests

## Suggested Checklist:

- [X] I have performed a self-review of my own code
- [X] I have commented my code, particularly in hard-to-understand areas
- [X] I have made corresponding changes to the documentation
- [ ] I have added Google Colab support for the newly added notebooks.
- [X] My changes generate no new warnings
- [X] I have added tests that prove my fix is effective or that my feature works
- [X] New and existing unit tests pass locally with my changes
- [X] I ran `make format; make lint` to appease the lint gods

### CODE REVIEW

**Comment by Aktsvigun** [comment]
Hi @logan-markewich , could you please review/approve?

**Comment by Aktsvigun** [comment]
@logan-markewich  @yimao  could you please review?

File: llama-index-integrations/embeddings/llama-index-embeddings-nebius/README.md
Comment (logan-markewich) [inline llama-index-integrations/embeddings/llama-index-embeddings-nebius/README.md:1]: Let's put an actual readme here. Describing the install (`pip install llama-index-embeddings-nebius`) and how to use it

File: llama-index-integrations/llms/llama-index-llms-nebius/Untitled.ipynb
Comment (logan-markewich) [inline llama-index-integrations/llms/llama-index-llms-nebius/Untitled.ipynb:1]: Lets remove this file

File: llama-index-integrations/llms/llama-index-llms-nebius/README.md
[the remark this answers is not in the document]
Comment (Aktsvigun) [reply llama-index-integrations/llms/llama-index-llms-nebius/README.md:1]: done

File: llama-index-integrations/llms/llama-index-llms-nebius/Untitled.ipynb
Comment (Aktsvigun) [reply llama-index-integrations/llms/llama-index-llms-nebius/Untitled.ipynb:1]: done

File: llama-index-integrations/embeddings/llama-index-embeddings-nebius/README.md
Comment (Aktsvigun) [reply llama-index-integrations/embeddings/llama-index-embeddings-nebius/README.md:1]: done

**Review by logan-markewich — APPROVED** [review]

**Comment by Aktsvigun** [comment]
@logan-markewich sorry for tagging you, guess this is the final effort required :)

**Pull request merged**